### PR TITLE
Feat: Run with correct npm path

### PR DESCRIPTION
**Pull request by Fabian-Fynn**

### CODE REVIEW

File: src/commands/start.command.ts [hunk 1]
```diff
@@ -9,7 +9,8 @@ export const start = async (apps: AppDefinition[]) => {
 
   function startProcess(app: AppDefinition) {
     console.log("starting " + app.script);
-    const p = spawn("bash", ["-c", app.script]);
+    const NPM_PATH = execSync("npm bin").toString().trim();
```
Comment (kaufmo) [inline src/commands/start.command.ts:12]: this should be optional or configurable as interpreter setting, npm can be default for example

Comment (nsams) [reply src/commands/start.command.ts:12]: you mean to be independent of npm? (yarn or even non-node environment)

**Comment by nsams** [comment]
I'm unsure if this feature is really needed, it's like when you are on the commandline, there you also have to use ./node_modules/.bin

**Comment by Fabian-Fynn** [comment]
I needed to add this to make some of our scripts work (wait-on, etc.)

**Review by nsams — APPROVED** [review]

File: src/commands/start.command.ts (hunk 1 above)
Comment (kaufmo) [reply src/commands/start.command.ts:12]: yes but this can be done later :+1:

**Pull request merged**